tests/node/rpc/messages/OnVehicleDataTests.js

```diff
@@ -32,12 +39,23 @@ describe('OnVehicleDataTests', function () {
             [GearStatus.KEY_TRANSMISSION_TYPE]: TransmissionType.MANUAL,
         };
 
+        this.climateData = new ClimateData()
```

Comment:
You can delete lines 42-50 (`this.climateData` and `JSON_CLIMATE_DATA`) because these would be defined in `Test.js`.

Reply:
Fixed in d1e6617